### ISSUE TITLE
Cover Image Block: Fixed background appears blown out and blurry in iOS

### ISSUE THREAD

**Issue by laurelfulford** [WordPress/gutenberg#4133]
## Issue Overview
On an iOS device, the fixed background appears blown out and blurry in the Cover Image block. Due to how iOS scales the image, the longer the page is, the worse it appears.

## Steps to Reproduce (for bugs)
1. Add a Cover Image block to a post.
2. Set the background to be fixed.
3. To compare, add a second Cover Image block to the same post. Do not set the background of this one to be fixed.

## Expected Behavior
The background images will scale the same on mobile and desktop.

## Current Behavior
The background images look as expected on desktop, but the fixed background one is super-zoomed in and blurry on iOS devices -- I'm specifically testing on an iPhone 6 running iOS 11.2.

## Possible Solution
I think it might be necessary to remove the fixed background on iOS devices to get around this (that's how it was handled in Twenty Seventeen) but there might be another way around it!

## Screenshots / Video
For comparison:

Here's a cover block, no fixed background:

![Without](https://cldup.com/AvN0NuF-ym-3000x3000.png)

And a cover block, using the exact same photo, with a fixed background:

![With](https://cldup.com/S82PG6qv2a-3000x3000.png)

## Related Issues and/or PRs
It seemed similar to #2600, so I originally added a comment there, but this issue's still persisting.

**Comment by aduth** [comment]
Likely related: https://stackoverflow.com/a/23420490/995445

**Comment by jeffpaul** [comment]
_This ticket was mentioned in [Slack](https://make.wordpress.org/chat/) in #core-editor by jeffpaul. [View the logs](https://wordpress.slack.com/archives/C02QB2JS7/p1521135150000288)._

**Comment by craigburgess** [comment]
What about just targeting iOS like this? I've implemented it like this on a client site and it sees to work well.

`    &.has-parallax {
        @supports (-webkit-overflow-scrolling: touch) {
            background-attachment: scroll;
        }
    }
`

**Comment by designsimply** [comment]
Tested with iPhone 6S running iOS 12.0.1 and confirmed that Cover image blocks with the Settings > Block > Cover Settings > "Fixed Background" toggled on results in an image a bit more zoomed in but not as completely zoomed and blurry as the originally reported screenshots above.

Cover block, fixed background, viewed in the editor:

<img src="https://user-images.githubusercontent.com/1119271/47683406-185a9600-db95-11e8-8c6b-b97c358974ef.PNG" width="270">

Cover block, fixed background, viewed on the front end:

<img src="https://user-images.githubusercontent.com/1119271/47683427-27414880-db95-11e8-8ca0-fe456b4ce57f.PNG" width="270">

<sup>Seen at http://alittletestblog.com/wp-admin/post.php?post=15377&action=edit and http://alittletestblog.com/2018/10/29/cover/ using WordPress 4.9.8 and Gutenberg 4.1.1 and iPhone 6S running iOS 12.0.1.</sup>

**Comment by mtias** [comment]
@designsimply should this be closed?

**Comment by mtias** [comment]
Thanks for the added context!

**Comment by dartiss** [comment]
I don't think this is limited to iOS. Viewing a cover image with fixed background in MacOS (using Chrome), I'm seeing quite a reduced resolution.

You can see a demonstration of this in this post - https://artiss.blog/2018/11/how-to-add-a-parallax-image-to-your-gutenberg-posts/.

**Comment by jasmussen** [comment]
@dartiss I'm not sure that is the same issue. The image you used for background image in that post, https://artiss.blog/wp-content/uploads/2018/02/IMG_0386-710x533.png, is quite low resolution already, so it will look blurry when stretched.

**Comment by dartiss** [comment]
Interesting 🤔That should have been a full size image - let me investigate some more. Apologies for the inconvenience.

**Comment by jasmussen** [comment]
No worries at all, thank you for the report, it landed in the right place!

**Comment by dartiss** [comment]
I duplicated the first block and it appears the lower resolution image appeared then but I can't recreate it now. I'll put this down as a one-off, as I've now corrected all the images in that post.
